notebooks/hawkdovevar_population_risk_category.ipynb

```diff
@@ -0,0 +1,1316 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.        if c in [9,10,11,12]:
</pre><p>This could be {9,10,11,12} for faster lookup. same for other lists</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Princeton-CDH/simulating-risk/pull/39/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>